gap-analysis/index.md

```diff
@@ -15,106 +15,127 @@ layout: wgnote
+<li> this narration text is focused on producing speech, and has different trade-offs when producing Braille. Generally, the common set of single characters is a lot less frustrating in Braille as there is a single cell corresponding to each character, which is brief and easy to traverse. In speech constructs such as “open paren” or “long left right arrow” can be a mouthful, especially in expressions with many uses of them.</li>
+</ul>
+</details>
 
-* $(0,5)$ -- Point, could be an open interval, gcd, cycle, or an ordered tuple, vector etc.
-* $\overline{A'B'}$   -- Line segment, could be mean (average), complex conjugate, or just simply “overbar”
-* $B'_x$ -- scripted variable
+</details>
 
-    * subscript: x-coordinate, subscripted-variable, index in a matrix, argument x of the function B′
-    * superscript: Prime, could be arcminutes, feet, derivative
+## The problem of ambiguity
 
-These will be used to illustrate different ways of potentially resolving the ambiguity in that notation.
+Common mathematical expressions such as $ax^2+bx+c=0$ are mostly unambiguous. However there are some notations that are ambiguous which makes interpretation difficult. As mentioned above $(a,b)$ could represent a point, or the open interval from ‘a’ to ‘b’, or the gcd of ‘a’ and ‘b’. There are several other potential interpretations. It is also the case that different notations can be used to represent the same concept. For example $]a,b[$ is also sometimes used to represent the open interval from ‘a’ to ‘b’ (but not the gcd, etc). These ambiguities present difficulties for AT that wishes to provide semantic speech.
 
+In general, math braille is presentational in that the braille describes the math that is seen, so problems with Presentation MathML are much fewer for braille generation.
+The MathML WG has identified three examples where braille is not presentational in Nemeth code (a common braille math code in the US and some other countries) inside of a mathematical expression.
+<details>
+<summary>
+Examples of semantic differences
+</summary>
+<ul>
+<li>The combinatorial symbol $\binom{m}{3}$  is easily confused with a 2x1 column vector. Note that Nemeth braille can express these two cases unambiguously, so we can promote them as “best practices”. The combinatorial for `\binom{n}{k}` is ⠷⠝⠩⠅⠾ and the matrix is ⠠⠷⠝⠠⠾⠠⠷⠅⠠⠾</li>
+<li>“:” either is a ratio (which has spaces on either side in Nemeth) or something else (mapping, field extension, …) which is prefixed with a punctuation indicator and has no spacing. Note there is a ratio code point in Unicode (U+2236), but its use is not common in MathML.</li>
+<li>Vertical Bar ( \| ) has many meanings. When used as a sign of comparison as defined by Nemeth code, it has spaces around, otherwise it doesn’t. Examples of a sign of comparison: $\lbrace x \mid x \in ℝ\rbrace$ and $P(A\vert B)$. Examples where it is not a sign of comparison: $\vert x\vert$ and $x\|3$ (x divides 3).</li>
```

Comment:
here and elsewhere markdown quoting such as `\|` is still in the html but as noted in email we can revert to using markdown within the details element so I propose to leave this and we can fix up as part of general edits after the merge.